Makefile

```diff
@@ -162,13 +162,15 @@ install: default
 	$(INSTALL) -d $(INST_LUADIR)/apisix/plugins/zipkin
 	$(INSTALL) apisix/plugins/zipkin/*.lua $(INST_LUADIR)/apisix/plugins/zipkin/
 
+
```

Comment:
why do you add this blank line? I think one blank is good here.

Reply:
one PR for one thing. Please do not update irrelevant files.

Reply:
I think it maybe merge problem, I will fix it

Reply:
> one PR for one thing. Please do not update irrelevant files.

I think it maybe merge problem, I will fix it

Reply:
> why do you add this blank line? I think one blank is good here.

got it

Reply:
fix it